frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-design/experiment-design.component.ts

```diff
@@ -617,6 +618,18 @@ export class ExperimentDesignComponent implements OnInit, OnChanges, OnDestroy {
   }
 
   toggleAliasTable(): void {
+    if (!this.isAliasTableDisplayed) {
+      this.stepContainer.nativeElement.scroll({
+        top: this.stepContainer.nativeElement.scrollHeight / 2,
+        behavior: 'smooth'
+      });
+    }
+    else {
+      this.stepContainer.nativeElement.scroll({
+        top: 0,
+        behavior: 'smooth'
+      });
+    }
     this.isAliasTableDisplayed = !this.isAliasTableDisplayed;
```

Comment:
`isAliasTableDisplyed` flag was probably only used for the `[hidden]` attribute in the html that you change removes, I think we can safely remove this line and it's declaration. would you be able to check and remove if so?

Reply:
`isAliasTableDisplayed` is still being used to toggle between the two pages, but I can also remove it and create 2 separate functions (e.g. `scrollToAliasesTable()` and `scrollToConditionsTable()`) instead of `toggleAliasTable()`. I can do it later.

Reply:
oh okay nah don't worry about that, I thought it was no longer used, nevermind!